From feb61dd5dc996c86960bd779004ff030ab727dc2 Mon Sep 17 00:00:00 2001
From: Kiran Kelageri <kirankelageri@codeaurora.org>
Date: Thu, 4 Oct 2018 17:52:31 -0700
Subject: [PATCH 09/37] Bluetooth-TWS: Disable outgoing connection to 2nd
 earbud

Disbaling outgoing connection to 2nd earbud and removed
vendor specific property.

 Conflicts:
	packages/SettingsLib/src/com/android/settingslib/bluetooth/CachedBluetoothDevice.java

Change-Id: I912c3acd84f449b6f967d6ef2c7beaebca4ef05c
CRs-fixed: 2326015
Signed-off-by: Volodymyr Zhdanov <wight554@gmail.com>
---
 .../android/settingslib/bluetooth/CachedBluetoothDevice.java  | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/packages/SettingsLib/src/com/android/settingslib/bluetooth/CachedBluetoothDevice.java b/packages/SettingsLib/src/com/android/settingslib/bluetooth/CachedBluetoothDevice.java
index 6ef9a3a02bc..04b84ab984c 100644
--- a/packages/SettingsLib/src/com/android/settingslib/bluetooth/CachedBluetoothDevice.java
+++ b/packages/SettingsLib/src/com/android/settingslib/bluetooth/CachedBluetoothDevice.java
@@ -29,7 +29,6 @@ import android.os.Looper;
 import android.os.Message;
 import android.os.ParcelUuid;
 import android.os.SystemClock;
-import android.os.SystemProperties;
 import android.text.TextUtils;
 import android.util.EventLog;
 import android.util.Log;
@@ -60,6 +59,7 @@ public class CachedBluetoothDevice implements Comparable<CachedBluetoothDevice>
     private static final long MAX_HEARING_AIDS_DELAY_FOR_AUTO_CONNECT = 15000;
     private static final long MAX_HOGP_DELAY_FOR_AUTO_CONNECT = 30000;
     private static final long MAX_MEDIA_PROFILE_CONNECT_DELAY = 60000;
+    private static final boolean mIsTwsConnectEnabled = false;
 
     private final Context mContext;
     private final BluetoothAdapter mLocalAdapter;
@@ -737,7 +737,7 @@ public class CachedBluetoothDevice implements Comparable<CachedBluetoothDevice>
         refresh();
 
         if (bondState == BluetoothDevice.BOND_BONDED) {
-            if (SystemProperties.getBoolean("persist.vendor.btstack.connect.peer_earbud", false)) {
+            if (mIsTwsConnectEnabled) {
                 Log.d(TAG, "Initiating connection to" + mDevice);
                 if (mDevice.isBondingInitiatedLocally() || mDevice.isTwsPlusDevice()) {
                     connect();
-- 
2.28.0

